frontend/src/app/home/electricity/page.tsx

```diff
@@ -28,21 +28,27 @@ export default function Electricity() {
   const [data, setData] = useState<ElectricityDataProps>([]);
 
   useEffect(() => {
-    async function fetchData() {
-      const temp = await fetch(
-        `http://${process.env.NEXT_PUBLIC_HOST_URL}/service/house/0/electricity`,
-        {
-          next: { revalidate: 60 }, // Revalidate every 60 seconds
-        },
-      );
-      setData(await temp.json());
+    const socket = new WebSocket('ws://localhost:8080/websocket');
```

Comment:
I think this is pretty good actually! However, replace `localhost` by `${process.env.NEXT_PUBLIC_HOST_URL}`, so that it works in the production machine :)